app/assets/javascripts/sip/geo.js.coffee

```diff
@@ -88,10 +88,11 @@
       $("#" + idcla).attr("disabled", true) if idcla
       $('#' + idcla).trigger('chosen:updated')
 
-
+@llena_departamento = ($this, root, sincoord=false) ->
+  llena_departamento_congancho($this, root, sincoord=false)
```

Comment:
Está perdiendo el valor que venga en sincoord al poner: llena_departamento_congancho($this, root, sincoord=false) debe ser algo como llena_departamento_congancho($this, root, sincoord, null, null)  y manejar los casos con los parametros finales en null en llena_departamento_congancho